internal/collector/event.go

```diff
@@ -106,15 +106,15 @@ func (collector *EventCollector) eventsTotalHandler() cache.ResourceEventHandler
 		AddFunc: func(obj interface{}) {
 			ev := obj.(*v1.Event)
 			// Count only Events created after the exporter starts running.
-			if beforeLatestEvent(startRunning, ev) {
+			if beforeOrEqualLatestEvent(startRunning, ev) {
 				collector.increaseEventsTotal(ev, 1)
 			}
 		},
 		UpdateFunc: func(oldObj, newObj interface{}) {
 			oldEv := oldObj.(*v1.Event)
 			newEv := newObj.(*v1.Event)
 			// Count only Events updated after the exporter starts running.
```

Comment:
Do we need to update this comment as well?

Reply:
Yes, I will update it and make it clearer that we are checking for reconciliation.